docs/src/main/sphinx/release/release-330.rst

```diff
@@ -39,11 +39,11 @@ Server
 Security
 --------
 
-* :ref:`principal_rules` are deprecated and will be removed in a future release.
+* :ref:`system-file-auth-principal-rules` are deprecated and will be removed in a future release.
```

Comment:
IIRC changes to older release notes don't reflect on the website unless explicitly deployed by someone? @martint would know.